spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/plugin/GravitinoDriverPlugin.java

```diff
@@ -75,6 +87,9 @@ private void registerGravitinoCatalogs(
               String catalogName = entry.getKey();
               Catalog gravitinoCatalog = entry.getValue();
               String provider = gravitinoCatalog.provider();
+              if ("lakehouse-iceberg".equals(provider) && supportsIceberg == false) {
```

Comment:
could you change `provider` to  `provider.toLowerCase(Locale.ROOT)` ?

Reply:
done